### PR TITLE
blog: add october 13th pre-announcement security release

### PR DESCRIPTION
**Pull request by RafaelGSS**
Refs: https://github.com/nodejs-private/node-private/issues/485

### CODE REVIEW

**Review by tniessen — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by rodion-arr — APPROVED** [review]

File: pages/en/blog/vulnerability/october-2023-security-releases.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+date: 2023-10-12:30:00.000Z
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/october-2023-security-releases.md]:
```suggestion
date: 2023-10-12T14:30:00.000Z
```
Looks like the date here is malformed?

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: site.json [hunk 2]
```diff
@@ -33,10 +33,10 @@
   ],
   "websiteBanners": {
     "index": {
-      "startDate": "2023-07-31T14:30:00.000Z",
-      "endDate": "2023-08-16:00:00.000Z",
-      "text": "Security releases now available",
-      "link": "https://nodejs.org/en/blog/vulnerability/august-2023-security-releases/"
+      "startDate": "2023-10-12T14:30:00.000Z",
+      "endDate": "2023-10-13:00:00.000Z",
```
Comment (MattIPv4) [inline site.json]: Ditto here, this timestamp seems funky?

File: pages/en/blog/vulnerability/october-2023-security-releases.md (hunk 1 above)
Comment (mhdawson) [inline pages/en/blog/vulnerability/october-2023-security-releases.md]:
```suggestion

date: 2023-10-12T13:30:00.000Z
```

Comment (mhdawson) [reply pages/en/blog/vulnerability/october-2023-security-releases.md]: just fixed that.

File: site.json (hunk 2 above)
Comment (mhdawson) [reply site.json]: looking at that one now.

Comment (mhdawson) [reply site.json]: well fixed the dates in header, fixing these ones now.

Comment (mhdawson) [inline site.json]:
```suggestion
      "startDate": "2023-10-12T13:30:00.000Z",
      "endDate": "2023-10-19T13:30:00.000Z",
```

File: pages/en/blog/vulnerability/october-2023-security-releases.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/october-2023-security-releases.md]:
```suggestion
```

**Comment by ovflowd** [comment]
@mhdawson you will need to add the label again

**Review by MattIPv4 — APPROVED** [review]

**Pull request merged**